docs/reference/contributing/guidelines/workflow.md

```diff
@@ -109,13 +107,13 @@ Time: 1 day for CI to complete and report back results.
 
 ##### Work needed
 
-A pull request in the work needed state requires additional work due to failed tests. If a pull request is in this state, our maintainers request changes from the pull request author.
+A pull request in the work needed state requires additional work due to failed tests or rework as a result of the review. If a pull request is in this state, our maintainers request changes from the pull request author.
 
-Time: 3 days for the pull request author to respond to the review comments.
+Time: 3 days for the pull request author to action to the review comments.
```

Comment:
you didn't remove the superfluous 'to'. Should say 'to action the review comments'